app/views/api/v2/key_pairs/main.json.rabl

```diff
@@ -0,0 +1,4 @@
+object @key_pair
+
+attributes :name, :active, :used_elsewhere
```

Comment:
maybe in_used vs used_elsewhere?
is there a way via the API / UI to figure out where its being used?